src/trunk.c

```diff
@@ -6183,19 +6187,16 @@ trunk_pivot_lookup(trunk_handle     *spl,
 // If any change is made in here, please make similar change in
 // trunk_lookup_async
 platform_status
-trunk_lookup(trunk_handle *spl, char *key, writable_buffer *data)
+trunk_lookup(trunk_handle *spl, char *key, merge_accumulator *data)
```

Comment:
I find `data` to be a confusing name. What about `lookup_result` or just `result`?